common_test.go

```diff
@@ -17,8 +18,16 @@ func unhex(h string) []byte {
 }
 
 func assert(t *testing.T, test bool, msg string) {
+	prefix := string("")
+	for i := 1; ; i++ {
+		_, file, line, ok := runtime.Caller(i)
+		if !ok {
+			break
+		}
+		prefix = fmt.Sprintf("%v: %d\n", file, line) + prefix
```

Comment:
Nit: Might be nicer to keep these to a single line, e.g., "file:line | file:line | file:line > msg"

Reply:
It's pretty hard to read that way it turns out. Will do if you insist.